components/portal/src/components/home/index.js

```diff
@@ -0,0 +1,287 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import AppBar from '@material-ui/core/AppBar';
+import BgImg from '../../img/celleryBg.png';
```

Comment:
Shall we rename this file to "celleryOverviewBg.png" ?
(Adding comment here since we can't add comments to binary files)